azurerm/internal/services/advisor/data_source_advisor_recommendations.go

```diff
@@ -59,7 +60,7 @@ func dataSourceArmAdvisorRecommendations() *schema.Resource {
 							Computed: true,
 						},
 
-						"recommendation_name": {
+						"recommendation_id": {
```

Comment:
This is a breaking change, we'll have to support both and deprecate the old property - why are we removing the name of the recomendation? seems valuable?

Reply:
I agree. I'll keep them both.